pyrcareworld/pyrcareworld/envs/rcareworld_env.py

```diff
@@ -290,7 +290,7 @@ def create_robot(
         urdf_path: str = None,
         base_pos: list = [0, 0, 0],
         base_orn=[-0.707107, -0.707107, -0.707107, 0.707107],
-    ) -> None:
+    ) -> Robot:
```

Comment:
Could you remove the class type constraints for the returned object for all of the functions in this file?

Reply:
Do you mean *add* the proper class type constraints? Currently, `Robot` is working and is super helpful since it allows us to autocomplete / type check functions it provides. `None` makes it so nothing autocompletes.

Reply:
I just pushed a commit to do this! [6b328a6](https://github.com/empriselab/RCareWorld/pull/41/commits/6b328a6e1c82b9ac9eda731b693d5da8cd20a12e)